colmena/proxy.py

```diff
@@ -0,0 +1,42 @@
+"""Utilities for interacting with ProxyStore"""
+import proxystore as ps
+
+from typing import Any, Union
+
+to_proxy = ps.to_proxy
+"""Place object in value server and return proxy
+
+Alias of `proxystore.to_proxy() <https://proxystore.readthedocs.io/en/latest/source/proxystore.html#proxystore.to_proxy>`_.
+"""
+
+
+def extract_and_evict(proxy: ps.proxy.Proxy) -> object:
+    """Extract wrapped object from proxy and evict from value server"""
```

Comment:
Could you fill in the Args for this docstring? I'm not sure where a proxy object comes from, etc

Reply:
Done